apps/designer/app/canvas/shared/use-shortcuts.ts

```diff
@@ -75,6 +79,7 @@ export const useShortcuts = () => {
     breakpointsMenu: publishOpenBreakpointsMenu,
     breakpoint: publishSelectBreakpoint,
     zoom: publishZoom,
+    cancel: publishCancelCurrentDrag,
```

Comment:
cancelDrag
```suggestion
    cancelDrag: publishCancelCurrentDrag,
```

Reply:
or alternatively just send a generic "esc" and let those subscribe who need to know ?